#ifndef KYRINBOX_SRC_SERVER_SLAVENODE_KYRIN_SLAVENODE_SYNC_H_
#define KYRINBOX_SRC_SERVER_SLAVENODE_KYRIN_SLAVENODE_SYNC_H_

namespace kyrin {
namespace server {

class KyrinSlavenodeSync {
public:
    bool start_sync();
};

} /* server */
} /* kyrin */

#endif /* end of include guard: KYRINBOX_SRC_SERVER_SLAVENODE_KYRIN_SLAVENODE_SYNC_H_ */
